express/src/controllers/clients.ts

```diff
@@ -15,6 +15,19 @@ export const showClient: RequestHandler = async (req, res) => {
     const serviceName = client.serviceName;
     const redirectUrls = String(client.redirectUris).split(",");
     const userPublicKey = client.publicKey == defaultPublicKey ? "" : getAuthApiCompliantPublicKey(client.publicKey);
+    const userAttributes = [];
+
+    if (client.scopes.includes("email")) {
+        userAttributes.push("Email address");
+    }
+
+    if (client.scopes.includes("phone")) {
+        userAttributes.push("Phone number");
+    }
+
+    if (client.scopes.includes("offline_access")) {
+        userAttributes.push("Offline access");
+    }
```

Comment:
```suggestion
    if (client.scopes.includes("email")) {
        userAttributes.push("Email address");
    }

    if (client.scopes.includes("phone")) {
        userAttributes.push("Phone number");
    }

    if (client.scopes.includes("offline_access")) {
        userAttributes.push("Offline access");
    }
```
The controller should return the original array. The view should decide what content to show for each token (e.g. print "Offline access" for the `offline_access` token)